backend/api/Services/MissionSchedulingService.cs

```diff
@@ -598,7 +575,14 @@ private bool TheSystemIsAvailableToRunAMission(Robot robot, MissionRun missionRu
                 return false;
             }
 
-            if (robot.Status is not RobotStatus.Available)
+            if (
+                !(
+                    robot.Status
+                    is RobotStatus.Available
+                        or RobotStatus.Docked
+                        or RobotStatus.ReturningHome
+                )
+            )
```

Comment:
Same as before, could also be a function on robot status:
`robot.Status.IsStatusWhereRobotAcceptNewMisison()`

Reply:
Same as before